external_search.py

```diff
@@ -50,6 +47,69 @@ def __init__(self, url=None, works_index=None):
         self.index = self.__client.index
         self.delete = self.__client.delete
         self.exists = self.__client.exists
+
+    def setup_index(self):
+        if self.works_index:
+            if self.indices.exists(self.works_index):
+                self.indices.delete(self.works_index)
+
+            self.log.info("Creating index %s", self.works_index)
+            self.indices.create(
+                index=self.works_index,
+                body={
+                    "settings": {
+                        "analysis": {
+                            "filter": {
+                                "en_stop_filter": {
+                                    "type": "stop",
+                                    "stopwords": ["_english_"]
+                                },
+                                "en_stem_filter": {
+                                    "type": "stemmer",
+                                    "name": "english"
+                                },
+                                "en_stem_minimal_filter": {
+                                    "type": "stemmer",
+                                    "name": "english"
+                                },
+                            },
+                            "analyzer" : {
+                                "en_analyzer": {
+                                    "type": "custom",
+                                    "char_filter": ["html_strip"],
+                                    "tokenizer": "standard",
+                                    "filter": ["lowercase", "asciifolding", "en_stop_filter", "en_stem_filter"]
+                                },
+                                "en_minimal_analyzer": {
+                                    "type": "custom",
+                                    "char_filter": ["html_strip"],
+                                    "tokenizer": "standard",
+                                    "filter": ["lowercase", "asciifolding", "en_stem_minimal_filter"]
+                                },
+                            }
+                        }
+                    }
+                },
+            )
+        
+            mapping = {"properties": {}}
+            for field in ["title", "series", "subtitle", "summary"]:
+                mapping["properties"][field] = {
```

Comment:
Flagging this as something I don't understand and would like to talk about in person. Why do you use both the en_analyzer and the en_minimal_analyzer here?